.rubocop.yml

```diff
@@ -8,8 +8,8 @@ AllCops:
     - 'dashboard/scripts/**/*'
     - 'dashboard/db/**/*'
     - 'dashboard/db/*'
-  RunRailsCops: true
   DisplayCopNames: true
+  TargetRubyVersion: 2.2
```

Comment:
Did this one line deletion and one line insertion propagate to all the changes below (ignoring the 12_345_678 fixes)?